### DISCUSSION TITLE
pen options

### DISCUSSION THREAD

**Discussion by i-make-robots** [MarginallyClever/Makelangelo-software#374] [Ideas]
pen size used to be a thing in rendering. because of the way opengl renders lines it was hard to get the correct line size even while zooming.
The Turtle class knows nothing about the line size, and the TurtleRenderer classes don't know about the Pen settings, so ... it's a question of how to bring these pieces of info together in a sane way.

**Comment by PPAC37** [comment]
Is there a definition of the tool / "Pen" in an instance of a machine and profiles linked to this tool? (as with the slicers of 3D printers there is a notion of profiles for the heads (the notion of nozzle size seems to overlap with the notion of "pen size"))

**Comment by PPAC37** [comment]
https://github.com/MarginallyClever/Makelangelo-software/blob/8fe5def5b94775d3ba8b3a75c70718d60a027356/src/main/com/marginallyclever/makelangelo/plotter/settings/PenSettingsPanel.java#L16
?
edit : why the reason of the  Deprecated  ?
